src/main/java/pl/edu/mimuw/BankClient.java

```diff
@@ -1,35 +1,67 @@
 package pl.edu.mimuw;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class BankClient {
+  private final String name;
+  private final String surname;
+  private final int age;
+  private final String accountNumber;
+  private double availableFunds;
```

Comment:
klient jest mutowalny - spoczko
jednak raczej jak w banku zwracasz liste kleintow, to zwroc ich kopie,  a nie tylko kopie listy